asdf_astropy/converters/unit/tests/test_quantity.py

```diff
@@ -4,9 +4,17 @@
 from asdf.testing import helpers
 from astropy import units
 from astropy.units import Quantity
+from astropy.utils.introspection import minversion
 from numpy.testing import assert_array_equal
 
 
+def asdf_open_memory_mapping_kwarg(memmap: bool) -> dict:
```

Comment:
The `bool` annotation here is leading to a pre-commit/ruff failure:
https://results.pre-commit.ci/run/github/271820376/1721652625.YDD78eseT3m3vTW-_PD9kw
https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/
I am ok with either of the following:
- a global ignore for all `FBT` rules.
- removing this type hint